modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/ErrorsModelFactory.java

```diff
@@ -132,9 +133,12 @@ private String getErrorNamespace(ErrorTypeDefinition errorType) {
   private void addType(ErrorTypeDefinition<?> errorType,
                        Graph<ErrorTypeDefinition, Pair<ErrorTypeDefinition, ErrorTypeDefinition>> graph) {
     graph.addVertex(errorType);
-    ErrorTypeDefinition parentErrorType = errorType.getParent().orElse(ANY);
-    graph.addVertex(parentErrorType);
-    graph.addEdge(errorType, parentErrorType);
+    String type = errorType.getType();
+    if (!ANY.name().equals(type) && !CRITICAL.name().equals(type)) {
```

Comment:
It messes up with the hierarchy, connectivity and retry are mapped to Mule's types, but any others to their own ANY. This makes it consistent. If we want a way to match all errors from an extension then we can add a special "namespace" matcher in the future. Today you could use the "when" attribute and use a custom expression.